interbotix_ros_xscobots/interbotix_xscobot_moveit/launch/xscobot_moveit.launch.py

```diff
@@ -299,9 +301,13 @@ def launch_setup(context, *args, **kwargs):
     )
 
     return [
-        move_group_node,
-        moveit_rviz_node,
         xscobot_ros_control_launch_include,
+        TimerAction(
+                period=5.0,
+                actions=[move_group_node,
+                         moveit_rviz_node,
+                        ],
+                    ),
```

Comment:
Strange indent again — make sure to run linters. I would also leave the TimerAction out of this PR. You can have it there locally for your machine, though.